core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.scss

```diff
@@ -58,3 +58,11 @@
     transform: translateY(-5px);
   }
 }
+
+.workflow-snapshot {
+  float: left;
+  display: inline;
+  border: 1px solid #343a40;
```

Comment:
can you please decrease the size of the window preview

Reply:
Changed to 160px width, 90px height